src/index.js

```diff
@@ -294,4 +294,35 @@ export default class Eik {
 			'Eik config was not loaded or "loadMaps" is "false" when calling .maps()',
```

Comment:
The `maps` function is documented as _The Way_ to generate the same HTML you now get from `toHTML`.

We have to update the docs to clarify which one you should use when.

- https://github.com/eik-lib/node-client/blob/c5e077500393010b47b1ef249f281a2bde8bf090/src/index.js#L270-L289
- https://github.com/eik-lib/node-client/blob/6738052917a3a7f71377ccc31cc500fb22743e79/README.md?plain=1#L73-L89

And on the docsite:
- https://github.com/eik-lib/eik-lib.github.io/blob/e2a109cc6e11c5545c6500b78e39a0bd002512c0/docs/guides/browser-importmap.md?plain=1#L83-L107
- https://github.com/eik-lib/eik-lib.github.io/blob/e2a109cc6e11c5545c6500b78e39a0bd002512c0/docs/reference/at-eik-node-client.md?plain=1#L75-L91